sdk/kusto/azure-mgmt-kusto/azure/mgmt/kusto/version.py

```diff
@@ -9,4 +9,4 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.7.0"
+VERSION = "0.7.1"
```

Comment:
versoin should be 0.8.0

Reply:
committed here: https://github.com/Azure/azure-sdk-for-python/pull/11774/commits/d52bf23e4bb0a39475a6f70061fb5bc7b451c921

Reply:
fixed here: https://github.com/Azure/azure-sdk-for-python/pull/11774/commits/5ca7343ae41581433cb56535cf2d00c54d8e7823